template/workflows/helm/draft.yaml

```diff
@@ -36,7 +36,7 @@ variables:
   - name: "CHARTOVERRIDES"
     default: 
       disablePrompt: true
-      value: "replicas:2"
+      value: "replicas=2,image.tag=${{ github.sha }}"
```

Comment:
this doesn't seem like the right place for this to me. I'd just put the image.tag part directly in the workflow. We don't want that part to be configurable, it's completely necessary

Reply:
Okay, I've added the image tag in the helm command itself